CONTRIBUTING.md

```diff
@@ -38,3 +38,131 @@ are conflicts (even trivially resolvable ones).
+- All contributions to the project source code ("patches") SHALL use the same license as the project.
+- All patches are owned by their authors. There SHALL NOT be any copyright assignment process.
+- The copyrights in the project SHALL be owned collectively by all its Contributors.
+- Each Contributor SHALL be responsible for identifying themselves in the project Contributor list.
+
+### Patch Requirements
+
+- Maintainers and Contributors MUST have a Platform account and SHOULD use their real names or a well-known alias.
+- A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.
+- A patch MUST adhere to the code style guidelines of the project if these are defined.
+- A patch MUST adhere to the "Evolution of Public Contracts" guidelines defined below.
+- A patch SHALL NOT include non-trivial code from other projects unless the Contributor is the original author of that code.
+- A patch MUST compile cleanly and pass project self-tests on at least the principle target platform.
+- A patch commit message SHOULD consist of a single short (less than 50 character) line summarizing the change, optionally followed by a blank line and then a more thorough description.
+- A "Correct Patch" is one that satisfies the above requirements.
+
+### Development Process
+
+- Change on the project SHALL be governed by the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
+- To request changes, a user SHOULD log an issue on the project Platform issue tracker.
+- The user or Contributor SHOULD write the issue by describing the problem they face or observe.
+- The user or Contributor SHOULD seek consensus on the accuracy of their observation, and the value of solving the problem.
+- Users SHALL NOT log feature requests, ideas, suggestions, or any solutions to problems that are not explicitly documented and provable.
+- Thus, the release history of the project SHALL be a list of meaningful issues logged and solved.
+- To work on an issue, a Contributor SHALL fork the project repository and then work on their forked repository.
+- To submit a patch, a Contributor SHALL create a Platform pull request back to the project.
+- A Contributor SHALL NOT commit changes directly to the project.
+- If the Platform implements pull requests as issues, a Contributor MAY directly send a pull request without logging a separate issue.
+- To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+- To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+- Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+- Maintainers SHALL NOT make value judgments on correct patches.
+- Maintainers SHALL merge correct patches from other Contributors rapidly.
```

Comment:
I want a delay so it can be reviewed without having to check github every day. Some reviews take days.